extra-libs/libfreeaptx/autobuild/patches/0001-make-the-makefile-compatible-for-autobuild3.patch

```diff
@@ -0,0 +1,40 @@
+diff -Naur libfreeaptx-0.1.1/Makefile libfreeaptx-0.1.1.aosc/Makefile
```

Comment:
`compatible-with` *

Reply:
Drop `the` as well.

Reply:
OK. I copied the file name from the `libopenaptx` corresponding patch. :P